README.md

```diff
@@ -36,17 +36,17 @@ As of 27th Jan 2022, we have also moved the UPnP functionality into this repo in
 
 Removing the UPnP container reduces CPU overhead and redundant code.
 
-## Helium Miner
+## Helium Gateway-rs
 
-Repo: [github.com/NebraLtd/hm-miner](https://github.com/NebraLtd/hm-miner)
+Repo: [github.com/NebraLtd/hm-gatewayrs](https://github.com/NebraLtd/hm-gatewayrs)
 
-This container is the actual Helium Miner software (from upstream), with the required configuration files added.
+This container is the actual Helium Gateway-rs software (from upstream), with the required configuration files added.
 
-## DBus Session
```

Comment:
this was the readme stuff I was mentioning in previous comment

Reply:
https://github.com/NebraLtd/helium-miner-software/pull/708#discussion_r1160661395